server/src/handlers.py

```diff
@@ -1,39 +1,94 @@
-import math
+from dataclasses import asdict
 
 from aiohttp import web
 from schematics.exceptions import DataError
+from asyncpg.exceptions import UniqueViolationError
 
 from src import db
-from src.models import room
-from src.utils import parse_json
+from src.models.user import User
+from src.models.paginator import Paginator
+from src.encrypt import encrypt_jwt
+from src.decorators import auth_required
+from src.status import GameStatus
 
 
 async def ping(request: web.Request) -> web.Response:
-    return web.json_response({'msg': 'pong'})
+    return web.json_response({'message': 'pong'})
 
 
-async def create_room(request: web.Request) -> web.Response:
+async def registration(request: web.Request) -> web.Response:
     data = await request.json()
-    some_room = room.Room(data)
+    user = User(data)
     try:
-        some_room.validate()
+        user.validate()
+        await db.create_user(request.app['pool'], user)
     except DataError as e:
         return web.json_response(e.to_primitive(), status=400)
-    if await db.room_is_occupied(data):
-        return web.json_response({'msg': 'room with that name already exists'}, status=409)
-    await db.create_room(data)
-    return web.json_response({'msg': 'OK'}, status=201)
-
-
-async def get_list_rooms(request: web.Request) -> web.Response:
-    page = int(request.query.get('page', '1'))
-    limit = int(request.query.get('limit', '10'))
-    rooms, total_count = await db.get_rooms(page, limit)
-    return web.json_response({
-        "rooms": rooms,
-        "paginator": {
-            "page": page,
-            "limit": limit,
-            "total_pages": math.ceil(total_count / limit),
-        }
-    }, status=200, dumps=parse_json)
+    except UniqueViolationError:
+        return web.json_response({'message': 'user with that name already exists'}, status=409)
+    return web.json_response({'message': 'OK'}, status=201)
+
+
+async def login(request: web.Request) -> web.Response:
+    data = await request.json()
+    remember = data.pop('remember', False)
+    user = User(data)
+    try:
+        user.validate()
+    except DataError as e:
+        return web.json_response(e.to_primitive(), status=400)
+    password = user.password
+    user = await db.get_user(request.app['pool'], user)
+    if user is None:
+        return web.json_response({'message': "user don't exists"}, status=404)
+    if not user.is_password_same(password):
```

Comment:
how about to create function in the encrypt module, `is_same_hash` and do the stuff of is_password_same function.
Why I think in this way.
1) User must don't know about other passwords
2) Purpos of user model - is to manipulate/validate user's data, compare hashes - is more about encrypting